src/libraries/System.Text.Json/source_generation/src/System.Text.Json.SourceGeneration.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <CLSCompliant>false</CLSCompliant>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);BUILDING_SOURCE_GENERATOR</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisCSharpWorkspacesVersion)" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" PrivateAssets="all" />
+
+    <PackageDestination Include="analyzers\dotnet\cs" />
```

Comment:
Did you reference this project from the PKGProj?

Reply:
I believe he did, he just broke down the runtimelab branch in separate commits so that it is easier to review.

Reply:
FWIW here is the change in the JsonCodeGen branch:
https://github.com/dotnet/runtimelab/blob/33815215fa7c2ac5c664159d651314c3d86aed17/src/libraries/System.Text.Json/pkg/System.Text.Json.pkgproj#L8

Reply:
I haven't done so yet. I'll reference it in the next commit.

Reply:
Sure, so long as it's not forgotten.  I couldn't tell if you were intentionally deferring that or not.